package.json at */Content/Scripts is missing 'ipc-event-emitter' dependency, cannot auto-resolve.

LogTemp: Warning: Your package.json at D:/Projects/UnrealEngine/MyFirstVrProject/Content/Scripts is missing 'ipc-event-emitter' dependency, cannot auto-resolve

```
PIE: Server logged in
PIE: Play in editor total start time 0.483 seconds.
LogUObjectGlobals: Warning: Failed to find object 'Class None.SIOJJsonObject'
LogUObjectGlobals: Warning: Failed to find object 'Class None.SIOJJsonObject'
SocketIO: SocketIO Connected with session: OTbkP-6JnyfXjYeqAAAA
SocketIO: SocketIO OTbkP-6JnyfXjYeqAAAA connected to namespace: /
LogTemp: Main script Connected.
LogTemp: Error: Script Error: internal/modules/cjs/loader.js:883
  throw err;
  ^
Error: Cannot find module 'ipc-event-emitter'
Require stack:
- D:\Projects\UnrealEngine\MyFirstVrProject\Content\Scripts\NeurosityApp.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (D:\Projects\UnrealEngine\MyFirstVrProject\Content\Scripts\NeurosityApp.js:21:13)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'D:\\Projects\\UnrealEngine\\MyFirstVrProject\\Content\\Scripts\\NeurosityApp.js'
  ]
}
LogTemp: Warning: Your package.json at D:/Projects/UnrealEngine/MyFirstVrProject/Content/Scripts is missing 'ipc-event-emitter' dependency, cannot auto-resolve.
LogSlate: Window 'Message Log' being destroyed
LogSlate: Window 'Message Log' being destroyed
LogSlate: Warning: Unable to rasterize '../../../Engine/Content/Editor/Slate/Starship/AssetIcons/GameStateBase_16.svg'. File could not be found
```

--- a/Content/Scripts/.gitignore
+++ b/Content/Scripts/.gitignore
@@ -1,0 +1,2 @@
+
+/node_modules

--- a/Content/Scripts/package.json
+++ b/Content/Scripts/package.json
@@ -13,6 +13,7 @@
   "license": "MIT",
   "dependencies": {
     "@neurosity/notion": "^5.6.0",
-    "dotenv": "^10.0.0"
+    "dotenv": "^10.0.0",
+    "ipc-event-emitter": "^2.0.2"
   }
 }

